rts/Lua/LuaUnsyncedRead.cpp

```diff
@@ -1072,6 +1073,17 @@ int LuaUnsyncedRead::GetFrameTimeOffset(lua_State* L)
 	return 1;
 }
 
+/*** Gets game time for drawing purposes
+ *
+ * Returns the game time, taking the interpolated draw frame into account.
+ *
+ * @treturn number game time in seconds
+ */
+int LuaUnsyncedRead::GetGameSecondsInterpolated(lua_State* L)
+{
+	lua_pushnumber(L, (gs->GetLuaSimFrame() + globalRendering->timeOffset) / GAME_SPEED);
```

Comment:
I disagree with GAME_SPEED in this case, this is supposed to count integer simframes and fractional draws in between. adding in GAME_SPEED conflates this.

Reply:
This is by no means a blocker, just a good to know thing.

Reply:
Agree, but the function's name refers to seconds, so makes sense.